Linux 32-bit builds are failing as of 1.31.0

Seems like we might be missing a 32-bit package somewhere in the install_deps section. Error from Travis:

```
error /home/travis/build/VSCodium/vscodium/vscode/node_modules/keytar: Command failed.
Exit code: 1
Command: prebuild-install || node-gyp rebuild
Arguments: 
Directory: /home/travis/build/VSCodium/vscodium/vscode/node_modules/keytar
Output:
/home/travis/build/VSCodium/vscodium/vscode/node_modules/node-abi/index.js:30
  throw new Error('Could not detect abi for version ' + target + ' and runtime ' + runtime + '.  Updating "node-abi" might help solve this issue if it is a new release of ' + runtime)
  ^
Error: Could not detect abi for version 3.1.2 and runtime electron.  Updating "node-abi" might help solve this issue if it is a new release of electron
    at getAbi (/home/travis/build/VSCodium/vscodium/vscode/node_modules/node-abi/index.js:30:9)
    at module.exports (/home/travis/build/VSCodium/vscodium/vscode/node_modules/prebuild-install/rc.js:64:12)
    at Object.<anonymous> (/home/travis/build/VSCodium/vscodium/vscode/node_modules/prebuild-install/bin.js:8:25)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@8.15.0 | linux | x64
gyp http GET https://atom.io/download/electron/v3.1.2/iojs-v3.1.2-headers.tar.gz
gyp http 200 https://atom.io/download/electron/v3.1.2/iojs-v3.1.2-headers.tar.gz
gyp http GET https://atom.io/download/electron/v3.1.2/SHASUMS256.txt
gyp http 200 https://atom.io/download/electron/v3.1.2/SHASUMS256.txt
gyp info spawn /opt/pyenv/shims/python2
gyp info spawn args [ '/home/travis/.nvm/versions/node/v8.15.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/home/travis/build/VSCodium/vscodium/vscode/node_modules/keytar/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/home/travis/.nvm/versions/node/v8.15.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/home/travis/.node-gyp/iojs-3.1.2/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/home/travis/.node-gyp/iojs-3.1.2',
gyp info spawn args   '-Dnode_gyp_dir=/home/travis/.nvm/versions/node/v8.15.0/lib/node_modules/npm/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/home/travis/.node-gyp/iojs-3.1.2/<(target_arch)/iojs.lib',
gyp info spawn args   '-Dmodule_root_dir=/home/travis/build/VSCodium/vscodium/vscode/node_modules/keytar',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.' ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
make: Entering directory `/home/travis/build/VSCodium/vscodium/vscode/node_modules/keytar/build'
  CXX(target) Release/obj.target/keytar/src/async.o
  CXX(target) Release/obj.target/keytar/src/main.o
  CXX(target) Release/obj.target/keytar/src/keytar_posix.o
  SOLINK_MODULE(target) Release/obj.target/keytar.node
/usr/bin/ld: cannot find -lsecret-1
/usr/bin/ld: cannot find -lgio-2.0
/usr/bin/ld: cannot find -lgobject-2.0
/usr/bin/ld: cannot find -lglib-2.0
collect2: error: ld returned 1 exit status
make: *** [Release/obj.target/keytar.node] Error 1
make: Leaving directory `/home/travis/build/VSCodium/vscodium/vscode/node_modules/keytar/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/travis/.nvm/versions/node/v8.15.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at emitTwo (events.js:126:13)
gyp ERR! stack     at ChildProcess.emit (events.js:214:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
gyp ERR! System Linux 4.4.0-101-generic
gyp ERR! command "/home/travis/.nvm/versions/node/v8.15.0/bin/node" "/home/travis/.nvm/versions/node/v8.15.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/travis/build/VSCodium/vscodium/vscode/node_modules/keytar
gyp ERR! node -v v8.15.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -16,7 +16,7 @@ node_js: "8"
 before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash
   - export PATH="$HOME/.yarn/bin:$PATH"
-  - ./install_deps.sh
+  - . install_deps.sh
   - . get_repo.sh
   - . check_tags.sh
 

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -4,17 +4,20 @@ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
   brew update
   brew install jq zip
 else
-  sudo apt-get update
-  sudo apt-get install libx11-dev libxkbfile-dev libsecret-1-dev fakeroot rpm jq
   if [[ "$BUILDARCH" == "ia32" ]]; then
     sudo dpkg --add-architecture i386
     sudo apt-get update
-    sudo apt-get install libc6-dev-i386 gcc-multilib g++-multilib
-    sudo apt-get install libx11-dev:i386 libxkbfile-dev:i386
+    sudo apt-get install -y fakeroot rpm jq
+    sudo apt-get install -y libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386 gcc-multilib g++-multilib
+    export CFLAGS=-m32
+    export CXXFLAGS=-m32
   elif [[ $BUILDARCH == "arm64" ]]; then
     # Use the default C / C++ compilers,
     # because some makefiles default to CC:=gcc:
     export CC=/usr/bin/cc
     export CXX=/usr/bin/c++
+  else
+    sudo apt-get update
+    sudo apt-get install libx11-dev libxkbfile-dev libsecret-1-dev fakeroot rpm jq
   fi
 fi

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -5,9 +5,10 @@ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
   brew install jq zip
 else
   if [[ "$BUILDARCH" == "ia32" ]]; then
+    sudo apt-get install -y build-essential gcc g++
     sudo dpkg --add-architecture i386
     sudo apt-get update
-    sudo apt-get install -y fakeroot rpm jq build-essential
+    sudo apt-get install -y fakeroot rpm jq
     sudo apt-get install -y libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386 gcc-multilib g++-multilib
     export CFLAGS=-m32
     export CXXFLAGS=-m32

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -4,12 +4,16 @@ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
   brew update
   brew install jq zip
 else
+  sudo apt-get update
+  sudo apt-get install -y fakeroot rpm jq
   if [[ "$BUILDARCH" == "ia32" ]]; then
     sudo dpkg --add-architecture i386
     sudo apt-get update
-    sudo apt-get upgrade
-    sudo apt-get install -y fakeroot rpm jq
-    sudo apt-get install -y gcc-multilib g++-multilib libglib2.0-dev:i386 libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386
+    sudo apt-get install -y gcc-multilib g++-multilib
+    sudo apt-get install -y \
+      libx11-dev:i386 \
+      libxkbfile-dev:i386 \
+      libsecret-1-dev:i386
     export CFLAGS=-m32
     export CXXFLAGS=-m32
     export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig
@@ -19,7 +23,6 @@ else
     export CC=/usr/bin/cc
     export CXX=/usr/bin/c++
   else
-    sudo apt-get update
     sudo apt-get install libx11-dev libxkbfile-dev libsecret-1-dev fakeroot rpm jq
   fi
 fi

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -11,6 +11,7 @@ else
     sudo apt-get install -y libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386 gcc-multilib g++-multilib
     export CFLAGS=-m32
     export CXXFLAGS=-m32
+    export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig
   elif [[ $BUILDARCH == "arm64" ]]; then
     # Use the default C / C++ compilers,
     # because some makefiles default to CC:=gcc:

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -5,11 +5,11 @@ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
   brew install jq zip
 else
   if [[ "$BUILDARCH" == "ia32" ]]; then
-    sudo apt-get install -y build-essential gcc g++
     sudo dpkg --add-architecture i386
     sudo apt-get update
     sudo apt-get install -y fakeroot rpm jq
-    sudo apt-get install -y libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386 gcc-multilib g++-multilib
+    sudo apt-get install -y gcc-multilib g++-multilib
+    sudo apt-get install -y libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386
     export CFLAGS=-m32
     export CXXFLAGS=-m32
     export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -9,7 +9,9 @@ else
     sudo apt-get update
     sudo apt-get install -y fakeroot rpm jq
     sudo apt-get install -y gcc-multilib g++-multilib
-    sudo apt-get install -y gir1.2-secret-1:i386 libglib2.0-dev:i386   libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386
+    sudo apt-get install -y gir1.2-glib-2.0:i386
+    sudo apt-get install -y gir1.2-secret-1:i386
+    sudo apt-get install -y libglib2.0-dev:i386 libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386
     export CFLAGS=-m32
     export CXXFLAGS=-m32
     export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -11,6 +11,7 @@ else
     sudo apt-get update
     sudo apt-get install -y gcc-multilib g++-multilib
     sudo apt-get install -y \
+      libgirepository-1.0-1:i386 \
       gir1.2-glib-2.0:i386 \
       libglib2.0-dev:i386 \
       gir1.2-secret-1:i386 \

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -9,6 +9,7 @@ else
     sudo apt-get update
     sudo apt-get install -y fakeroot rpm jq
     sudo apt-get install -y gcc-multilib g++-multilib
+    sudo apt-get install -y libgirepository-1.0-1:i386
     sudo apt-get install -y gir1.2-glib-2.0:i386
     sudo apt-get install -y gir1.2-secret-1:i386
     sudo apt-get install -y libglib2.0-dev:i386 libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -7,12 +7,9 @@ else
   if [[ "$BUILDARCH" == "ia32" ]]; then
     sudo dpkg --add-architecture i386
     sudo apt-get update
+    sudo apt-get upgrade
     sudo apt-get install -y fakeroot rpm jq
-    sudo apt-get install -y gcc-multilib g++-multilib
-    sudo apt-get install -y libgirepository-1.0-1:i386
-    sudo apt-get install -y gir1.2-glib-2.0:i386
-    sudo apt-get install -y gir1.2-secret-1:i386
-    sudo apt-get install -y libglib2.0-dev:i386 libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386
+    sudo apt-get install -y gcc-multilib g++-multilib libglib2.0-dev:i386 libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386
     export CFLAGS=-m32
     export CXXFLAGS=-m32
     export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -7,7 +7,7 @@ else
   if [[ "$BUILDARCH" == "ia32" ]]; then
     sudo dpkg --add-architecture i386
     sudo apt-get update
-    sudo apt-get install -y fakeroot rpm jq
+    sudo apt-get install -y fakeroot rpm jq build-essential
     sudo apt-get install -y libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386 gcc-multilib g++-multilib
     export CFLAGS=-m32
     export CXXFLAGS=-m32

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -9,7 +9,7 @@ else
     sudo apt-get update
     sudo apt-get install -y fakeroot rpm jq
     sudo apt-get install -y gcc-multilib g++-multilib
-    sudo apt-get install -y libglib2.0-dev:i386 libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386
+    sudo apt-get install -y gir1.2-secret-1:i386 libglib2.0-dev:i386   libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386
     export CFLAGS=-m32
     export CXXFLAGS=-m32
     export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -9,7 +9,7 @@ else
     sudo apt-get update
     sudo apt-get install -y fakeroot rpm jq
     sudo apt-get install -y gcc-multilib g++-multilib
-    sudo apt-get install -y libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386
+    sudo apt-get install -y libglib2.0-dev:i386 libx11-dev:i386 libxkbfile-dev:i386 libsecret-1-dev:i386 libc6-dev-i386
     export CFLAGS=-m32
     export CXXFLAGS=-m32
     export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig

--- a/install_deps.sh
+++ b/install_deps.sh
@@ -11,6 +11,9 @@ else
     sudo apt-get update
     sudo apt-get install -y gcc-multilib g++-multilib
     sudo apt-get install -y \
+      gir1.2-glib-2.0:i386 \
+      libglib2.0-dev:i386 \
+      gir1.2-secret-1:i386 \
       libx11-dev:i386 \
       libxkbfile-dev:i386 \
       libsecret-1-dev:i386

